src/components/Homepage/Carousel/Item.astro

```diff
@@ -0,0 +1,58 @@
+---
+import { Image } from "astro:assets";
+import tickIcon from "../../../assets/icons/tick.svg";
+import "./ItemFrame.css";
+
+interface Props {
+  type?: "text" | "image";
+  message?: string;
+  source: string;
+  imageSrc?: ImageMetadata;
+  imageAlt?: string;
+  bottomWidth?: number;
+  padding?: number;
+}
+
+const {
+  type = "text",
+  message = "",
+  source,
+  imageSrc,
+  imageAlt = "",
+  bottomWidth = 30,
+  padding = 40,
+}: Props = Astro.props;
+---
+
+<div class="relative pb-3">
+  <div
+    class="pixelated-frame pt-3 text-center mx-auto bg-black text-white font-semibold font-mono text-sm"
+    style={`--bottom-segment-width: ${bottomWidth}px; padding-left: ${padding}px; padding-right: ${padding}px;`}
+  >
+    <!-- Frame parts -->
+    <div class="frame-part top-bar"></div>
+    <div class="frame-part top-right-corner"></div>
+    <div class="frame-part bottom-right-corner"></div>
+    <div class="frame-part bottom-left-segment"></div>
+    <div class="frame-part bottom-right-segment"></div>
+    <div class="frame-part bottom-left-corner"></div>
+    <div class="frame-part top-left-corner"></div>
+    <div class="frame-part left-bar"></div>
+    <div class="frame-part right-bar"></div>
+
+    <!-- Banner content based on type -->
+    {
+      type === "image" && imageSrc && (
+        <div class="py-1">
+          <Image src={imageSrc} alt={imageAlt} />
+        </div>
+      )
```

Comment:
If you apply my suggestion above, this will be simplified as follows:

```suggestion
      type === "image" && (
        <div class="py-1">
          <Image src={imageSrc} alt={imageAlt} />
        </div>
      )
```